app/_hub/kong-inc/statsd-advanced/index.md

```diff
@@ -28,56 +36,70 @@ kong_version_compatibility:
         - 1.5.x
         - 1.3-x
         - 0.36-x
-        - 0.35-x
-        - 0.34-x
 
 params:
   name: statsd-advanced
-  api_id: true
+  api_id: false
   service_id: true
   route_id: true
   consumer_id: true
   protocols: ["http", "https", "grpc", "grpcs", "tcp", "tls", "udp"]
   config:
     - name: host
-      required:
+      required: true
       default: "`127.0.0.1`"
       value_in_examples: 127.0.0.1
-      description: The IP address or host name of StatsD server to send data to
+      datatype: string
+      description: The IP address or hostname of StatsD server to send data to.
     - name: port
-      required:
+      required: true
       default: "`8125`"
       value_in_examples: 8125
-      description: The port of StatsD server to send data to
+      datatype: integer
+      description: The port of StatsD server to send data to.
     - name: metrics
-      required:
-      default: "All metrics<br>are logged"
+      required: true
+      default: "All metrics are logged"
+      datatype: Array of record elements
```

Comment:
```suggestion
      datatype: array of record elements
```

Reply:
@HCloward this is sentence case throughout plugin docs when it's more than one word. Can revisit this style though.